### ISSUE TITLE
Nonzero Treatment Start

### ISSUE THREAD

**Issue by drbergman** [laurenpmarazzi/PhysiPKPD#8]
Allow for user to specify when the first treatment begins rather than the default of 0.

Stretch goal: have the option to specify some condition to wait for before starting therapy. E.g. x% confluence. Ask Paul about this.

**Comment by drbergman** [comment]
User can specify a start time now. Will work on getting the simple confluence check of (total cell volume)/(total area) > threshold?

**Comment by drbergman** [comment]
Confluence check is in! Based on area. See merge comments.
